Userland/Libraries/LibWeb/Painting/BorderPainting.cpp

```diff
@@ -24,10 +24,14 @@ BorderRadiusData normalized_border_radius_data(Layout::Node const& node, Gfx::Fl
 
     // Scale overlapping curves according to https://www.w3.org/TR/css-backgrounds-3/#corner-overlap
     auto f = 1.0f;
-    f = min(f, rect.width() / (float)(top_left_radius_px + top_right_radius_px));
-    f = min(f, rect.height() / (float)(top_right_radius_px + bottom_right_radius_px));
-    f = min(f, rect.width() / (float)(bottom_left_radius_px + bottom_right_radius_px));
-    f = min(f, rect.height() / (float)(top_left_radius_px + bottom_left_radius_px));
+    auto width_reciprocal = 1.0f / rect.width();
+    auto height_reciprocal = 1.0f / rect.height();
+    f = max(f, width_reciprocal * (top_left_radius_px + top_right_radius_px));
+    f = max(f, height_reciprocal * (top_right_radius_px + bottom_right_radius_px));
+    f = max(f, width_reciprocal * (bottom_left_radius_px + bottom_right_radius_px));
+    f = max(f, height_reciprocal * (top_left_radius_px + bottom_left_radius_px));
+
+    f = 1.0f / f;
```

Comment:
How does this related to the 2048 target increment/decrement?

Reply:
This does not relate to the issue, at least not as far as I am aware of. The only change I was intentionally trying to make, was the change to the `Userland/Games/2048/GameSizeDialog.cpp` file, where I simply removed the ` - 1` from the end of line 19.

I am unaware of any other changes, so I would respectfully ask that the other two files that appear to be changed in this PR are declined.



I made the change and submitted this PR from my desktop Windows 10 PC which is using WSL, rather than my laptop which is using Ubuntu. If WSL and/or Windows 10 is somehow causing issues, from now on I will ensure that I do not use that computer to make any changes that would be submitted to the project. I apologize, as I am trying to get more familiar with git for everyday use. I suspect that I accidentally caused these two other file changes, but I'm not quite sure what would have caused it. (_Other than the likely possibility that I also messed something up on the WSL/SerenityOS installation on that Windows 10 machine._)

Reply:
> I am unaware of any other changes, so I would respectfully ask that the other two files that appear to be changed in this PR are declined.

That's not how PRs work. Everything you committed will be merged, or nothing.

Reply:
Understood! I will fix this, and create a new PR. I will now close this PR.

Before I submit my next PR, I will also to my best to see if I can figure out what was actually causing the increment/decrement issue with the target tile setting. (_If we're all spending this much time trying to fix this, but going back and forth, why not fix it to the best of my ability with a little more effort and understanding._) Thank you.